conf/app.routes

```diff
@@ -1,5 +1,6 @@
 # microservice specific routes
 
-GET        /hello-world             uk.gov.hmrc.trackingconsentfrontend.controllers.HelloWorldController.helloWorld
+->         /govuk-frontend          govuk.Routes
```

Comment:
I don't really understand what this does? Have you got some docs somewhere about govuk frontendy stuff?

Reply:
This defines some routes for the govuk frontend assets: https://github.com/hmrc/play-frontend-govuk